lbrytv/middleware/cache-control.js

```diff
@@ -0,0 +1,15 @@
+const THREE_MONTHS_IN_SECONDS = 7776000;
+
+async function redirectMiddleware(ctx, next) {
+  const {
+    request: { url },
+  } = ctx;
+
+  if (url.includes('.png') || url.includes('font.css') || url.includes('.woff')) {
```

Comment:
This check should be stronger so that it only catches the website assets. Remember that `file.png` is a prefectly valid claim name.

If we're not moving assets to a CDN off of the webserver, let's put all assets in a clearly named path so we can perform a strong check.

Reply:
Also, if asset names are unique to the asset (i.e. rather than font.css, we call it font-v5.css, or font-abc123hash.css), then the cache can be extremely long (essentially indefinite).